x-pack/test/screenshot_creation/apps/response_ops_docs/stack_cases/details_view.ts

```diff
@@ -6,28 +6,61 @@
  */
 
 import { FtrProviderContext } from '../../../ftr_provider_context';
+/*
+import { createAndUploadFile } from '../../../../cases_api_integration/common/lib/api';
+import { superUser } from '../../../../cases_api_integration/common/lib/authentication/users';
+import { CASES_FILE_KIND } from '../../../../cases_api_integration/common/lib/constants';
+*/
 
 export default function ({ getService }: FtrProviderContext) {
   const cases = getService('cases');
   const commonScreenshots = getService('commonScreenshots');
+  const testSubjects = getService('testSubjects');
 
   const screenshotDirectories = ['response_ops_docs', 'stack_cases'];
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
   let CASE_ID: string;
+  let CASE_OWNER: string;
 
   describe('details view', function () {
     before(async () => {
-      const { id: caseId } = await cases.api.createCase({
+      const testCase = await cases.api.createCase({
         title: 'Web transactions',
         tags: ['e-commerce'],
         description: 'Investigate e-commerce sample data.',
       });
-      CASE_ID = caseId;
+      CASE_ID = testCase.id;
+      CASE_OWNER = testCase.owner;
+      /*
+      await createAndUploadFile({
+        supertest: supertestWithoutAuth,
+        createFileParams: {
+          name: 'testfile',
+          kind: CASES_FILE_KIND,
+          mimeType: 'text/plain',
+          meta: {
+            caseIds: [CASE_ID],
+            owner: [CASE_OWNER],
+          },
+        },
+        data: 'abc',
+        auth: { user: superUser, space: null },
```

Comment:
It's probably failing here because `superUser` wasn't created. Could you try using this block of code:

```
const supertest = getService('supertest');
...
await createAndUploadFile({
        supertest,
        createFileParams: {
          name: 'testfile',
          kind: CASES_FILE_KIND,
          mimeType: 'text/plain',
          meta: {
            caseIds: [CASE_ID],
            owner: [CASE_OWNER],
          },
        },
        data: 'abc',
      });
```

By omitting the `auth` field and using the regular `supertest` hopefully it'll use the built in test users for the request.

Reply:
Yay, that worked, thanks!